CMakeLists.txt

```diff
@@ -88,8 +88,14 @@ set(CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE}
 # to the corresponding path in the source directory.
 function(link_to_source base_name)
     # Get OS dependent path to use in `execute_process`
-    file(TO_NATIVE_PATH "${CMAKE_CURRENT_BINARY_DIR}/${base_name}" link)
-    file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/${base_name}" target)
+    if (CMAKE_HOST_UNIX)
```

Comment:
I think it's better to use `CMAKE_HOST_WIN32` and change the order of the statements

Reply:
Done. Thanks.

Reply:
@geesun I don't see this was done. Please confirm